File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        int i=0, x=0, y=0, len=quiz.length;
        String[] answer = new String[len];
        String[][] split = new String[len][5];
        
        for(String q:quiz){
            split[i] = q.split(" ");
        
            x = Integer.parseInt(split[i][0]);
            y = Integer.parseInt(split[i][2]);
            if(split[i][1].equals("+")){
                answer[i] = Integer.toString(x+y);
            }else{
                answer[i] = Integer.toString(x-y);
            }
            answer[i] = answer[i].equals(split[i][4]) ? "O" : "X" ;
            i++;
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.js
function solution(quiz) {
    var answer = [];
    quiz.map(str => {
        var arr = str.split(" ");
        var x = Number(arr[0]);
        var y = Number(arr[2]);
        var z = Number(arr[4]);
        if(arr[1]=="+" && x+y===z) answer.push("O");
        else if(arr[1]=="-" && x-y===z) answer.push("O");
        else answer.push("X");
    })
    return answer;
}